core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -171,7 +171,7 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
     const ckConfig: RteConfig = {
       extraPlugins: 'pre,sharedspace,' + pluginNames,
       startupFocus: true,
-      removePlugins: 'indentblock',
+      removePlugins: 'contextmenu,tabletools,tableselection,indentblock',
```

Comment:
Hey @shricodev, would you please explain this code change? 
After you reply, feel free to assign me back to the PR. Thanks!

Reply:
Hello @SanjaySajuJacob, To fix the issue, the `contextmenu` plugin had to be removed from CKEditor. However, since the `tabletools` and `tableselection` plugins depend on `contextmenu`, they had to be removed as well. I hope this helps, Thanks!